### DISCUSSION TITLE
Reduce stdout logs

### DISCUSSION THREAD

**Discussion by JonathanBarrows** [Altinity/clickhouse-backup#767] [Q&A] [answered]
I am getting a ton of stdout INFO logs showing up in my GCP log explorer each time the backup is running the snapshot portion. This is a `kubectl exec` command that creates the backup, sends it to a GCP bucket and produces logs that look likes the below message. Is there a way to keep helpful logs entries such as failures, but removes this garbage? I believe I could use `> /dev/null 2>&1` but that will remove any helpful logs as well.

`[A 79.07 KiB / 79.07 KiB  100.00% 0s`

**Comment by Slach** [comment, marked as answer]
`/etc/clickhouse-backup/config.yml`
```
general:
  disable_progress_bar: true
```

Moreover, you could use `DISABLE_PROGRESS_BAR=true` environment variable
